portfolio/src/main/webapp/style.css

```diff
@@ -41,6 +41,36 @@ h3 {
   margin-bottom: 5px;
 }
 
+/* Add a black background color to the top navigation */
+.topnav {
```

Comment:
nit: maybe `top-navigation-bar` would be a better name?

Reply:
I agree, thanks!

Reply:
I agree, thanks!